test/consul_helpers.go

```diff
@@ -79,15 +90,20 @@ func runConsulClusterTest(t *testing.T, packerBuildName string, examplesFolder s
 		terraform.InitAndApply(t, terraformOptions)
 	})
 
-	test_structure.RunTestStage(t,"validate", func() {
+	test_structure.RunTestStage(t, "validate", func() {
 		awsRegion := test_structure.LoadString(t, exampleFolder, SAVED_AWS_REGION)
 		terraformOptions := test_structure.LoadTerraformOptions(t, exampleFolder)
+		keyPair := test_structure.LoadEc2KeyPair(t, exampleFolder)
 
-		// Check the Consul servers
-		checkConsulClusterIsWorking(t, CONSUL_CLUSTER_EXAMPLE_OUTPUT_SERVER_ASG_NAME, terraformOptions, awsRegion)
+		if len(enterpriseUrl) > 0 {
+			checkEnterpriseInstall(t, CONSUL_CLUSTER_EXAMPLE_OUTPUT_SERVER_ASG_NAME, terraformOptions, awsRegion, sshUser, keyPair)
```

Comment:
Shouldn't the enterprise install *also* check the cluster is working?

Reply:
done